releasenotes/notes/21715.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+issue:
+- 25744
+releaseNotes:
+- |
+  **Added** support to verify installation
```

Comment:
```suggestion
  **Added** support for post-install verification.

```

Reply:
Maybe provide more details in the release note about how to run the post-install verification command.

Reply:
Added some details.